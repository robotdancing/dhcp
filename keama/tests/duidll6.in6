# LL server duid config

# authoritative is mandatory
authoritative;

# LL server duid declaration
server-duid ll;

