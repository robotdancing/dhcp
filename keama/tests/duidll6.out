{
  # LL server duid config
  # authoritative is mandatory
  "Dhcp6": {
    # LL server duid declaration
    "server-id": {
      "type": "LL"
    }
  }
}
